### DISCUSSION TITLE
[macOS] [Config] Keybind: ctrl+space combination doesn't trigger pending sequence menu.

### DISCUSSION THREAD

**Discussion by maranix** [ghostty-org/ghostty#3481] [Feature Requests, Ideas] [answered]
Hey, I just installed the terminal after learning about the v1.0 release and so far I have been liking it. However, I've run into an issue while configuring the keybindings. For some reason the following keybind `ctrl+space>/=toggle_quick_terminal` doesn't trigger the pending sequence popup on the bottom. However changing `space` into `n` somehow does works. Is `space` key handled under a different keyword for keybindings?

Also `keybind = ctrl+n=clear_screen` doesn't seem to clear the terminal screen as well.

Config:
```text
# Font
font-size = 16
font-thicken = true

# Mouse & Cursor
mouse-hide-while-typing = true
cursor-style = block
cursor-style-blink = true

# Window
fullscreen = true
window-vsync = false
window-padding-x = 0
window-padding-y = 0

# Keybinds
# keybind = ctrl+;=clear_screen                                   # does not work
keybind = ctrl+space>/=toggle_quick_terminal           # doesn't work either
keybind = ctrl+n>/=toggle_quick_terminal                   # This does

# Quick Terminal
quick-terminal-position = bottom
```

**Comment by mitchellh** [comment]
Convert to discussion for issue triage.

**Reply by maranix** [reply, marked as answer]
@mitchellh This discussion can be closed.

Confirmed working on 1.0.1 release.

**Comment by astr0gator** [comment]
Guys, a short follow up.
Can I use ctrl+space as non-leader? 

I.e.
 # keybind = ctrl+space=next_tab

Doesn't work for me now.
